### DISCUSSION TITLE
Event for out of date/sync pull request

### DISCUSSION THREAD

**Discussion by chris-hinds** [community/community#24399] [Apps, API and Webhooks]
<p>Does GitHub current fire an event when a PR goes out ofdate/sync with its upstream branch? </p>
<p>Looking to find a way to automatically update branches where possible without continuously polling the api?</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>No, there isn’t a webhook or event that occurs like what you’re describing.</p>
<p>Let us know if you have more questions.</p>

**Comment by wei** [comment]
<p>Unfortunately no webhooks. My Github App polls periodically (hourly) and syncs master branch automatically.</p>
<p><a href="https://github.com/wei/pull" rel="nofollow noopener">https://github.com/wei/pull</a></p>
<p>:robot: a GitHub App built with probot that keeps your repository up-to-date with upstream changes via automated PRs.</p>
